deps/graph-parser/src/logseq/graph_parser/block.cljs

```diff
@@ -592,6 +592,22 @@
                 (assoc :block/updated-at updated-at))]
     (dissoc block :title :body :anchor)))
 
+(defn fix-duplicate-id
+  [block]
+  (-> block
+      (assoc :uuid (d/squuid))
+      (update :properties dissoc :id)
+      (update :properties-text-values dissoc :id)
```

Comment:
Good point about doing this. Unrelated to this PR we should fix :collapsed which doesn't do this

Reply:
Sorry, I don't get the part about fixing :collapsed.

Reply:
Let's address this issue in another PR.

Reply:
No worries. It's a one-liner so just pushed to master - https://github.com/logseq/logseq/commit/d7d65a6c6616baf17d1e24df3d80624f199eeb26